src/plugins/ucc_discovery_plugin.cpp

```diff
@@ -1,22 +1,42 @@
 #include "ucc_discovery_plugin.hpp"
 
+#include <algorithm>
+#include <cstddef>
+#include <cstdint>
+#include <functional>
+#include <memory>
+#include <optional>
+#include <sstream>
+#include <string>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
 #include <boost/container_hash/hash.hpp>
 
+// NOLINTNEXTLINE(misc-include-cleaner): We access methods of AbstractBenchmarkItemRunner in `pre_benchmark_hook()`.
 #include "../benchmarklib/abstract_benchmark_item_runner.hpp"
```

Comment:
would `IWYU pragma: keep` do anything here? Not 100% what it does, just stumbled upon it

Reply:
`pragma: keep` would be a good fit here (it just tells the include-cleaner to not complain about an include that "looks" unused), but in this case, the longer comment is very helpful. And from the documentation I don't see that comments are supported with pragmas (haven't tested though).

Reply:
Just stumbled upon is since I saw the following line in `.clang-format`:
https://github.com/hyrise/hyrise/blob/ebe0fabff633577230a1182668faaf03adaf4b7c/.clang-format#L40

Feel free to resolve